stable/spring-cloud-data-flow/templates/skipper-deployment.yaml

```diff
@@ -67,27 +67,36 @@ spec:
           value: /etc/secrets
         - name: SPRING_CLOUD_KUBERNETES_CONFIG_NAME
           value: {{ template "scdf.fullname" . }}-skipper
-        {{- if .Values.rabbitmq.enabled}}
+        {{- if or .Values.rabbitmq.enabled (index .Values "rabbitmq-ha" "enabled") }}
         - name: RABBITMQ_PASSWORD
           valueFrom:
             secretKeyRef:
+              {{- if index .Values "rabbitmq-ha" "enabled" }}
+              name: {{ printf "%s-%s" .Release.Name "rabbitmq-ha" | trunc 63 | trimSuffix "-" }}
+              {{- else }}
               name: {{ printf "%s-%s" .Release.Name "rabbitmq" | trunc 63 | trimSuffix "-" }}
+              {{- end }}
               key: rabbitmq-password
-        {{- end}}
+        {{- end }}
       volumes:
-        {{- if .Values.rabbitmq.enabled}}
+        {{- if or .Values.rabbitmq.enabled (index .Values "rabbitmq-ha" "enabled") }}
         - name: rabbitmq
           secret:
+            {{- if index .Values "rabbitmq-ha" "enabled" }}
+            secretName: {{ printf "%s-%s" .Release.Name "rabbitmq-ha" | trunc 63 | trimSuffix "-" }}
+            {{- else }}
             secretName: {{ printf "%s-%s" .Release.Name "rabbitmq" | trunc 63 | trimSuffix "-" }}
-        {{- end}}
+            {{- end }}
+        {{- end }}
         - name: mysql
           secret:
             secretName: {{ printf "%s-%s" .Release.Name "mysql" | trunc 63 | trimSuffix "-" }}
       initContainers:
-      - name: init-mysql-wait
-        image: {{ .Values.mysql.busybox.image }}:{{ .Values.mysql.busybox.tag }}
-        imagePullPolicy: {{ .Values.mysql.imagePullPolicy }}
-        command: ['sh', '-c', 'until nc -w3 -z {{ printf "%s-%s" .Release.Name "mysql" | trunc 63 | trimSuffix "-" }} 3306; do echo waiting for mysql; sleep 3; done;']
+      - name: init-db-wait
+        image: {{ .Values.initContainers.dbWait.image }}:{{ .Values.initContainers.dbWait.tag }}
+        imagePullPolicy: {{ .Values.initContainers.dbWait.imagePullPolicy }}
+        command: ['sh', '-c', 'until nc -w3 -z {{ template "scdf.database.host" . }} {{ template "scdf.database.port" . }}; do echo waiting for mysql; sleep 3; done;']
```

Comment:
maybe "waiting for database" rather than "waiting for mysql" as the external DB could be something else?